invenio_records_resources/services/errors.py

```diff
@@ -110,3 +110,9 @@ def __init__(self, max_files, resulting_files_count):
                 )
             )
         )
+
+
+class RecordSubmissionClosedCommunityError(PermissionDenied):
+    """Record submission policy forbids non-members from submitting records to community."""
+
+    description = "Submission to this community is only allowed to community members."
```

Comment:
minor: `invenio-records-resources` is too low in the dependency chain to know  about communities. Shouldn't this be in a module further up the chain?

Reply:
+1 for this, there shouldn't be any references to communities in the records-resources modules, since communities depends on records-resources, not the other way around

Reply:
Moved changes to https://github.com/inveniosoftware/invenio-rdm-records/pull/1790
closing this PR